src/azure-cli/azure/cli/command_modules/extension/__init__.py

```diff
@@ -70,6 +70,7 @@ def load_arguments(self, command):
             c.argument('extension_name', completer=extension_name_from_index_completion_list)
             c.argument('source', options_list=['--source', '-s'], help='Filepath or URL to an extension', completer=FilesCompleter())
             c.argument('yes', options_list=['--yes', '-y'], action='store_true', help='Do not prompt for confirmation.')
+            c.argument('version', default='latest', help='The specific version of an extension')
```

Comment:
Is there a way to get all available versions for specific extension?

Reply:
It's planned in @fengzhou-msft features. He will take care of it.